force-app/main/default/connectedApps/HOT_NAV_Tolkebestilling_android.connectedApp-meta.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ConnectedApp xmlns="http://soap.sforce.com/2006/04/metadata">
+    <contactEmail>vegard.aas.mjelde@nav.no</contactEmail>
+    <label>HOT_NAV_Tolkebestilling_android</label>
+    <oauthConfig>
+        <callbackUrl>https://login.salesforce.com/services/oauth2/callback</callbackUrl>
+        <consumerKey>3MVG904d7VkkD2aN8yOOjQM9wu4Sx1MFuBQaoLK_xc6yE.r0nbF25PHJcDbL0.VFPOwbS4rgOv.LT90uF5sEn</consumerKey>
+        <isAdminApproved>false</isAdminApproved>
```

Comment:
Burde denne være true? Eventuelt samme på den for iOS.

`Admin approved users are pre-authorized`

Da kan man spesifisere at at f.eks. et PS gir tilgang.

Reply:
Ok. Done